geoai/universat.py

```
@@ -0,0 +1,209 @@
+    "UniverSatProcessor",
+    "universat_inference",
+    "get_tile_embedding",
+    "get_pca_rgb",
+    "universat_train",
+]
+
+# Auto-clone UniverSat repo and setup environment
+_path = os.path.expanduser("~/.cache/geoai/UniverSat")
+_src = os.path.join(_path, "src")
+if not os.path.exists(_path):
+    subprocess.check_call(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)